src/bicep/mlz.bicep

```diff
@@ -252,6 +252,58 @@ module sharedServicesVirtualNetworkPeering './modules/spokeNetworkPeering.bicep'
   }
 }
 
+module hubPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: 'policyAssignement'
+  scope: resourceGroup(hubSubscriptionId, hubResourceGroupName)
+  dependsOn: [
+    hubResourceGroup
+    logAnalyticsWorkspace
+  ]
+  params: {
+    builtInAssignment: policy
+    logAnalyticsWorkspaceName: logAnalyticsWorkspace.outputs.name
+    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroupName
+  }
+}
+
+module operationsPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: '${operationsResourceGroupName}-policyAssignment'
+  scope: resourceGroup(operationsSubscriptionId, operationsResourceGroupName)
+  params: {
+    builtInAssignment: policy
+    logAnalyticsWorkspaceName: logAnalyticsWorkspace.outputs.name
+    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroup.name
+  }
+}
+
+module sharedServicesPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: 'policyAssignement'
+  scope: resourceGroup(sharedServicesSubscriptionId, sharedServicesResourceGroupName)
+  dependsOn: [
+    sharedServicesResourceGroup
+    logAnalyticsWorkspace
+  ]
+  params: {
+    builtInAssignment: policy
+    logAnalyticsWorkspaceName: logAnalyticsWorkspace.outputs.name
+    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroupName
```

Comment:
```suggestion
    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroup.outputs.name
```

Reply:
Interesting this has an issue when `scope: resourceGroup(operationsSubscriptionId, operationsResourceGroupName)` works. Guess scoping is different and it worked in test deployments. My goal was not to have dependency on other bicep so it can be run independently. I've made the changes for now but will revert when I can run more tests.

Reply:
You can still target the individual modules/policyAssignment module with whatever resource group you want to deploy with ad-hoc, but in this file, mlz.bicep, we have a dependency on the operationsResourceGroup existing to apply policy to it. There is no error because the Log Analytics Workspace resource today has that dependency too so you'd be safe, but, accessing this operationsResourceGroup.outputs object explicitly states the dependency rather than order of operations from some other resource or module.